sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DropTableExec.scala

```diff
@@ -27,7 +26,6 @@ import org.apache.spark.sql.connector.catalog.{Identifier, Table, TableCatalog}
  * Physical plan node for dropping a table.
  */
 case class DropTableExec(
```

Comment:
Can we avoid changing it? It's also being changed in https://github.com/apache/spark/pull/30491

Reply:
OK, will revert.